File: Mobile/miclink/lib/features/connection/data/miclink_protocol.dart
class MicLinkProtocol {
  MicLinkProtocol._();

  static const int version = 1;
  static const int heartbeatIntervalMs = 3000;
  static const int discoveryIntervalMs = 4000;

  static const String pairRequest = 'pair_request';
  static const String pairResponse = 'pair_response';
  static const String reconnectRequest = 'reconnect_request';
  static const String reconnectResponse = 'reconnect_response';
  static const String heartbeat = 'heartbeat';
  static const String heartbeatAck = 'heartbeat_ack';
  static const String muteUpdate = 'mute_update';
}

File: Pc/MicLinkWinUI/MicLinkWinUI/Infrastructure/Network/Protocol/MicLinkProtocol.cs
namespace MicLinkWinUI.Infrastructure.Network.Protocol;

public static class MicLinkProtocol
{
    public const int Version = 1;
    public const int HeartbeatIntervalMs = 3000;

    public static class MessageTypes
    {
        public const string PairRequest = "pair_request";
        public const string PairResponse = "pair_response";
        public const string ReconnectRequest = "reconnect_request";
        public const string ReconnectResponse = "reconnect_response";
        public const string Heartbeat = "heartbeat";
        public const string HeartbeatAck = "heartbeat_ack";
        public const string MuteUpdate = "mute_update";
    }
}
